Automattic/wp-calypso
# Magic Login: Migrate Image to Webpack

#### Changes proposed in this Pull Request

This should be a simple migration of the `/login` image folder, since it's only used once in Magic Login, added in #16813. (cc @danhauk)

For details on why this is necessary, see #36850

#### Testing instructions

Visit `/log-in/link` and request an email login link, then verify the image is still there.

<img width="666" alt="Screenshot 2019-10-16 at 17 08 37" src="https://user-images.githubusercontent.com/43215253/66937934-333d4900-f038-11e9-851e-39078e92b075.png">

This is the only usage of this image: https://github.com/Automattic/wp-calypso/search?q=check-email.svg&unscoped_q=check-email.svg
